public/app/features/alerting/unified/components/AlertTypeSection.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { FC, useState, useEffect } from 'react';
+import { GrafanaTheme, SelectableValue } from '@grafana/data';
+import { FieldSet, Field, Input, InputControl, stylesFactory, Select, FormAPI } from '@grafana/ui';
+import { config } from 'app/core/config';
+import { css } from 'emotion';
+
+import { getAllDataSources, getPromAndLokiDataSources } from '../utils/config';
+import { fetchRulerRules } from '../api/ruler';
+
+interface Props extends FormAPI<any> {}
+
+enum ALERT_TYPE {
+  THRESHOLD = 'threshold',
+  SYSTEM = 'system',
+  HOST = 'host',
+}
+
+const alertTypeOptions: SelectableValue[] = [
+  {
+    label: 'Threshold',
+    value: ALERT_TYPE.THRESHOLD,
+    description: 'Metric alert based on a defined threshold',
+  },
+  {
+    label: 'System or application',
+    value: ALERT_TYPE.SYSTEM,
+    description: 'Alert based on a system or application behavior. Based on Prometheus.',
+  },
+  {
+    label: 'Host',
```

Comment:
I don't think we'll have this to start with? Not in Figma anyway

Reply:
Yeah this was a holdover from the miro mockups and I don't think this will be included :)